airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py

```diff
@@ -1567,4 +1558,4 @@ class Quotes(CRMObjectIncrementalStream):
     entity = "quote"
     associations = ["deals"]
     primary_key = "id"
-    scopes = {"e-commerce"}
+    scopes = {"crm.objects.quotes.read", "crm.objects.deals.read"}
```

Comment:
Added scope for deals in this resource as it is mentioned under associations. @nidhilashkari17 Looks good?

Reply:
yes